### PR TITLE
Enable BFloat16 ELU, SELU and CELU in CPU path

### PR DESCRIPTION
**Pull request by yanbing-j**
Enable BFloat16 ELU, SELU and CELU in CPU path. SELU and CELU will call ELU implementation.

### CODE REVIEW

**Comment by yanbing-j** [comment]
@VitalyFedyunin Hi, could you please help review this PR? Thanks.

**Comment by frank-wei** [comment]
please rebase

**Comment by yanbing-j** [comment]
> please rebase

Hi @frank-wei , rebase is done. Thank you!

**Review by frank-wei — APPROVED** [review]

File: aten/src/ATen/native/cpu/Activation.cpp [hunk 1]
```diff
@@ -168,64 +168,136 @@ static void threshold_kernel(
...
-          if (!cmp.zero_mask()) {  // only a * poscoef (which is very quick) needs to be computed
-            return a * poscoef_vec;
-          } else {
-            return Vec::blendv((a * negiptcoef_vec).expm1() * negcoef_vec, a * poscoef_vec, cmp);
-          }
-        });
-  });
+      it,
+      [negcoef, negiptcoef, poscoef](BFloat16 a) -> BFloat16 {
+        return float(a) <= float(0) ? (std::exp(float(a) * negiptcoef) - float(1)) * negcoef : float(a) * poscoef;
```
Comment (swolchok) [inline aten/src/ATen/native/cpu/Activation.cpp:183]: why did we stop using expm1?

**Pull request closed without merging**